frameworks/cassandra/universe/config.json

```diff
@@ -83,6 +83,11 @@
           ],
           "default": "serial"
         },
+        "region": {
```

Comment:
From a customer UX perspective, would it make sense to have the "region" field and the "rack" and "data_center" fields adjacent to each other, as they both have to do with placement of the service. This is of course something that can be changed after the fact.